### DISCUSSION TITLE
NWJS

### DISCUSSION THREAD

**Discussion by 01ingsit1** [Blinue/Magpie#856] [Q&A] [answered]
When upscaling a modern NWJS app, it's upscaled with the top border included, like this: 

![nwjs](https://github.com/Blinue/Magpie/assets/87073345/f9d991f7-307b-4864-bee7-7beee03001f7)

I know I can just crop it, but still, is it working how it is supposed to? I'm not sure, so I put my question here. Sorry if me being dumb. Also, I've tried different capture modes and white theme, it doesn't change anything.

**Comment by Blinue** [comment, marked as answer]
Please refer to #268. NW.js draws its own title bar instead of using the system one, which means that other programs cannot determine its title bar height. The custom cropping functionality was added to support such windows.

**Reply by 01ingsit1** [reply]
Got it, thanks!
